publication-model/src/main/resources/publication-ontology.ttl

```diff
@@ -961,79 +961,79 @@ nva:DegreeLicentiate a rdfs:Class ;
   rdfs:subClassOf nva:Degree .
 
 nva:AllowedFileOperation a rdfs:Class ;
-  rdfs:comment "A thing that indicates what you can do with a file" ;
+  rdfs:comment "A permitted action that the user can apply to the file" ;
```

Comment:
nitpicking: Perhaps `"An operation that the user is authorized to perform on a File."`?